samples/bluetooth/peripheral_lbs/README.rst

```diff
@@ -3,22 +3,88 @@
 Bluetooth: Peripheral LBS
 #########################
 
+This sample demonstrates how to use the :ref:`lbs_readme`.
+
 Overview
 ********
 
-This sample demonstrates how to use the :ref:`lbs_readme`. When connected
-the example will send the state of button 1 on the development board
-to the mobile application. While the mobile application can control the
-state of the LED 3 on development board.
+The peripheral LBS sample demonstrates how to use the :ref:`lbs_readme`. When connected, the example sends the state of button 1 on the development board to the connected device (a phone or tablet). The mobile application on the device can display the received button state and can control the state of the LED 3 on the development board.
 
 Requirements
 ************
 
-* Any Nordic development board
-* A mobile running a compatible application (nRF Connect, nRF Blinky, nRF Toolbox)
+* One of the Nordic development boards:
+
+  * nRF52840 Development Kit board (PCA10056)
```

Comment:
nrf51 dongle and nrf52840 dongle?
I think they should work, the have 1 button and color LED instead of number

Reply:
Adding the dongle support may be a good idea, but I would like to keep it out of this PR which just touches documentation. I created NCSDK-2183